src/main/java/iroha/validation/rules/Rule.java

```diff
@@ -1,5 +1,18 @@
 package iroha.validation.rules;
 
+import iroha.protocol.TransactionOuterClass;
+
+/**
+ * Rule interface
+ */
 public interface Rule {
 
+  /**
+   * Method for checking transaction rule satisfiability
+   *
+   * @param transaction Iroha proto transaction
+   * @return <code>true</code> if the transaction satisfies rule;
+   * <code>false</code> otherwise
+   */
+  Boolean isSatisfiedBy(TransactionOuterClass.Transaction transaction);
```

Comment:
Why `Boolean`? Why not primitive `boolean`?

Reply:
Done